hibernate/src/test/java/com/example/spring/orm/HibernateTests.java

```diff
@@ -0,0 +1,14 @@
+package com.example.spring.orm;
+
+import org.junit.jupiter.api.Test;
+import org.springframework.boot.test.context.SpringBootTest;
+
+@SpringBootTest
+class HibernateTests {
+
+	@Test
+	void contextLoads() {
```

Comment:
Can be removed. Every test which is in `src/main/test` now executes on the JVM and in a native image.

Reply:
removed